Test/Tests/Network/IPFabric.Tests.ps1

```diff
@@ -88,17 +127,17 @@ Test-WithRetries 3 {
         BeforeEach {
             $BeforeEachStack = $Testenv.NewCleanupStack()
             $BeforeEachStack.Push(${function:Merge-Logs}, @(, $Testenv.LogSources))
-            $BeforeEachStack.Push(${function:Remove-AllContainers}, @(, $Testenv.Sessions))
+            $BeforeEachStack.Push(${function:Remove-AllContainers}, @(, $Testenv.Testbeds))
 
             Write-Log "Creating container: $ContainerID"
             New-Container `
-                -Testbed $Testenv.Sessions[0] `
+                -Testbed $Testenv.Testbeds[0] `
                 -NetworkName $VirtualNetwork.Name `
                 -Name $ContainerID `
-                -Image $DockerImage
+                -Image $Testbed.DefaultDockerImage
```

Comment:
can't you just not pass this param?

Reply:
I can, forgot to change when introduced this option.